style.css

```diff
@@ -174,3 +135,27 @@ footer {
 .not-read {
     background-color: rgb(80, 20, 20);
 }
+
+.active {
+    display: block;
+}
+
+.disappear {
+    transform: scale(0);
+    transition: 0.3s;
+}
+
+.appear {
+    transform: scale(1);
+    transition: 0.3s;
+}
```

Comment:
You can do just `.disappear` and apply it to an element by toggling it in JS, this way CSS will automatically revert to the base styling without having to explicitly code a "reverse-styling" classname. `transition` stylings should always be declared in the element base styling so other stylings applied to that element can be affected by it:

```css
.element {
  background: red;
  transition: all 0.3s ease-in;
}

.fill-blue {
  background: blue;
}
```